File: FrontEnd/SoftPac/SoftPacPersistance/DAO/CuentasAcreedorDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface CuentasAcreedorDAO
    {
        int Insertar(CuentasAcreedorDTO cuentaAcreedor);
        int Modificar(CuentasAcreedorDTO cuentaAcreedor);
        int eliminar(CuentasAcreedorDTO cuentaAcreedor);
        int eliminarLogico(CuentasAcreedorDTO cuentaAcreedor);
        CuentasAcreedorDTO ObtenerPorId(int cuentaAcreedorId);
        IList<CuentasAcreedorDTO> ObtenerPorAcreedor(int acreedorId);
        IList<CuentasAcreedorDTO> ListarTodos();
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/CuentasAcreedorDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.ArrayList;
import java.util.List;
import pe.edu.pucp.softpac.model.CuentasAcreedorDTO;

public interface CuentasAcreedorDAO {
    public Integer insertar(CuentasAcreedorDTO cuentaAcreedor);
    
    public CuentasAcreedorDTO obtenerPorId(Integer cuentaAcreedorId);
    
    public List<CuentasAcreedorDTO> listarTodos();
    
    public Integer modificar(CuentasAcreedorDTO cuentaAcreedor);
    
    public Integer eliminar(CuentasAcreedorDTO cuentaAcreedor);

    public Integer eliminarLogico(CuentasAcreedorDTO cuentaAcreedor);

    public ArrayList<CuentasAcreedorDTO> obtenerPorAcreedor(Integer acreedor_id);
}
